packages/ember-glimmer/lib/syntax/curly-component.js

```diff
@@ -290,9 +290,11 @@ class CurlyComponentManager {
   }
 
   didCreate({ component }) {
-    component.trigger('didInsertElement');
-    component.trigger('didRender');
-    component._transitionTo('inDOM');
+    if (false) {
```

Comment:
Hehe, this was just for testing on the node side.  Will remove before landing...